client/state/current-user/gravatar-status/test/actions.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	GRAVATAR_UPLOAD_RECEIVE,
+	GRAVATAR_UPLOAD_REQUEST,
+	GRAVATAR_UPLOAD_REQUEST_SUCCESS,
+	GRAVATAR_UPLOAD_REQUEST_FAILURE
+ } from 'state/action-types';
+import { uploadGravatar } from '../actions';
+import useNock from 'test/helpers/use-nock';
+import useFakeDom from 'test/helpers/use-fake-dom';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+describe( 'actions', () => {
+	let sandbox, spy;
+	useFakeDom();
```

Comment:
Ooh, yes, that is a great idea. I removed `useFakeDom` and stubbed FormData instead.